sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
+
+- Added the optional `serviceVersion` property to the `KeyClient` and `CryptographyClient` optional parameters.
+  It defaults to the latest supported API version, which currently is `7.1`.
+- Added `import` to the list of possible values for `KeyOperation`.
+- Added `recoverableDays` as an optional property to `KeyProperties`.
```

Comment:
For both the above points, can we add a line describing either how this is useful to the user, or which operation uses the `KeyOperation` or `KeyProperties`?
Same for `recoverableDays ` added to `SecretProperties` and `CertificateProperties`

Since JS users don't have to type their input or output to methods (and TS users have the same options), they wont be as familiar with interface names as they would be class methods. So it would help if add some more details on where this new thing that we have added actually surfaces to the user. We don't have to provide a complete list if it touches multiple methods. But providing even one or two instances should be enough for the reader to do the connection on their own

Reply:
Here's what I'm adding:

For certificates:

```md
- Added `recoverableDays` as an optional property to `KeyProperties`.
    - The Key Vault Keys client `4.1.0` now allows for Keys to be created with a determined number of days in which they can be recovered after deletion.
      For this reason, the Certificates client will now show this `recoverableDays` property when reading the key in use. This is only relevant for Azure Key Vaults with the soft-delete setting enabled.
```

For keys:

```md
- Added `recoverableDays` as an optional property to `KeyProperties`.
    - Keys can now be created with a determined number of days in which they can be recovered after deletion.
      This is only relevant for Azure Key Vaults with the soft-delete setting enabled.
```

For secrets:

```md
- Added `recoverableDays` as an optional property to `SecretProperties`.
    - Secrets can now be created with a determined number of days in which they can be recovered after deletion.
      This is only relevant for Azure Key Vaults with the soft-delete setting enabled. 
```